packages/react-native-cli/src/commands/AutomateSymbolicationCommand.ts

```diff
@@ -38,15 +38,16 @@ export default async function run (projectRoot: string, urls: OnPremiseUrls): Pr
     const { androidIntegration } = await prompts({
       type: 'confirm',
       name: 'androidIntegration',
-      message: 'Do you want to automatically upload JavaScript source maps as part of the Gradle build?',
+      message: 'Do you want to install the BugSnag CLI to allow you to upload JavaScript source maps?',
```

Comment:
I think that's probably TMI for the user at this point. All they probably care about it the outcome:

```suggestion
      message: 'Do you want to automatically upload JavaScript source maps for Android builds?',
```

Reply:
@tomlongridge is this still the wording that we want to go with ?